ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java

```diff
@@ -327,7 +327,7 @@ void fetchAndRunCommands() {
 
   @SuppressFBWarnings("DMI_INVOKING_TOSTRING_ON_ARRAY")
   private void executeStatement(final QueuedCommand queuedCommand) {
-    final String commandId = queuedCommand.getCommandId().toString();
+    final String commandId = queuedCommand.getAndDeserializeCommandId().toString();
```

Comment:
@pgaref can we remove the `DMI_INVOKING_TOSTRING_ON_ARRAY` FB suppression now?

Reply:
Thats why I commited this in the first place, and cherry-picking from 7.2 tricked me! Thanks :D